prover/zkevm/prover/ecdsa/adress.go

```diff
@@ -236,11 +236,13 @@ func (addr *Addresses) assignMainColumns(
 	split := splitAt(nbEcRecover)
 	n := nbRowsPerEcRec
 
-	var (
-		hashHi, hashLo, isHash, trimmedHi []field.Element
-	)
-
 	permTrace := keccak.GenerateTrace(pkModule.Data.ScanStreams(run))
+
```

Comment:
Given the number of consecutive allocations, it would make sense to wrap all that in a var statement

Reply:
I don't think I understand this. Now `:=` shorthand is used for declaring and initializing. Why is `var` necessary`

Reply:
It's not. It just looks nicer as it indents everything. I'll pass on that one as it's not super important either